cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -270,6 +270,8 @@ type AutoscalingOptions struct {
 	MinReplicaCount int
 	// NodeDeleteDelayAfterTaint is the duration to wait before deleting a node after tainting it
 	NodeDeleteDelayAfterTaint time.Duration
+	// NodeDeleteDelayAfterTaintUpdateTimestamp it the last update timestamp of the NodeDeleteDelayAfterTaint
+	NodeDeleteDelayAfterTaintUpdateTimestamp time.Time
```

Comment:
Why is this in `AutoscalingOptions`? It isn't something user would configure. What they might want to configure instead though is whether the dynamic logic is enabled at all, instead of using a fixed interval.